spec/system/casa_admins/edit_spec.rb

```diff
@@ -106,13 +106,14 @@
   end
 
   it "can convert the admin to a supervisor", js: true do
-    another = create(:casa_admin)
-    visit edit_casa_admin_path(another)
+    visit edit_casa_admin_path(admin)
 
-    click_on "Change to Supervisor"
+    page.find("a", text: "Change to Supervisor").click
 
-    expect(page).to have_text("Admin was changed to Supervisor.")
-    expect(User.find(another.id)).to be_supervisor
+    aggregate_failures do
```

Comment:
I believe, this might not be a good approach to use this block in a production spec. It's better to use in local environment for testing.
We should not be running the user DB query at line#115, if the line#114's expectation gets failed.

Reply:
Got it. I'll update the spec.